### DISCUSSION TITLE
Use 24-hour clock when UI is in English

### DISCUSSION THREAD

**Discussion by tml1024** [Traewelling/traewelling#2038] [Ideas / Feature Requests]
Surely most of the users are in Europe anyway, and 24-hour clock would be the normal to use, especially in a train travel context, even if you use the site in English.

**Comment by MrKrisKrisu** [comment]
Probably the display of the time should rather be removed from the language files and rather offer a setting option for the time.

Since the 12 hour clock is used in most English-speaking countries, I would not simply change it across the board. There we probably need another solution. Are there any more opinions on this?

**Comment by tml1024** [comment]
The 12 hour clock might be used in everyday speech context in many English-speaking regions, but for railway timetables in Europe (which is what Traewelling covers), 24-hour clock has been used also in English for a long time. The European Rail Timetable uses the 24-hour clock, as did the Cook's Continental Timetable. nationalrail.co.uk uses the 24-hour clock.

But sure, it could be an option.

**Comment by garridinsi** [comment]
I think that we could detect the usage based on JS, and also add a toggle on the user settings so this could be customized.
